scalar.proto.ts

```diff
@@ -61,6 +63,7 @@ export type LedgerValidationRequest = {
   setCertHolderId: (certHolderId: string) => void;
   setCertVersion: (certVersion: number) => void;
   setSignature: (signature: Uint8Array) => void;
+  serializeBinary: () => Uint8Array;
```

Comment:
This is a mistake when I split the code into PRs.
(It should be included in the previous PR.